testnets/pocket/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "pocket-beta",
```

Comment:
Cannot have a dash in the chain_name property here, and must match the directory name exactly (should be "pockettestnet"). You could instead use Pocket Beta for the `pretty_name`